libs/providers/flipt/src/lib/flipt-provider.ts

```diff
@@ -169,7 +169,7 @@ export class FliptProvider implements Provider {
         };
       }
 
-      if (!variantEvaluation.match) {
+      if (!variantEvaluation.match && variantEvaluation.reason != 'DEFAULT_EVALUATION_REASON') {
```

Comment:
Is it okay if the reason is "TARGETING_MATCH" when using the default value defined in Flipt?

Reply:
@beeme1mr it follows the same pattern as `resolveBooleanEvaluation`.

Also the official OpenFeature documentation says
- `DEFAULT` - no dynamic evaluation occurred or dynamic evaluation yielded no result
- `TARGETING_MATCH` - the result of a dynamic evaluation, such as a rule or specific user-targeting

Is there something I'm missing? I don't see any conflict here.

Reply:
There's no conflict. I was just confirming that this was the intended behavior.

Reply:
@beeme1mr thank you!